sway/tree/layout.c

```diff
@@ -168,25 +168,36 @@ void container_move_to(struct sway_container *container,
 	struct sway_container *old_parent = container_remove_child(container);
 	container->width = container->height = 0;
 	container->saved_width = container->saved_height = 0;
-	struct sway_container *new_parent;
+
+	struct sway_container *new_parent, *new_parent_focus;
+	struct sway_seat *seat = input_manager_get_default_seat(input_manager);
+
+	// Get the focus of the destination before we change it.
+	new_parent_focus = seat_get_focus_inactive(seat, destination);
 	if (destination->type == C_VIEW) {
 		new_parent = container_add_sibling(destination, container);
 	} else {
 		new_parent = destination;
 		container_add_child(destination, container);
 	}
+
 	wl_signal_emit(&container->events.reparent, old_parent);
```

Comment:
Nitpick: Group the `wl_signal_emit` line with the block above (since that block does the reparenting) and move the blank line below.